h2o-core/src/test/java/water/fvec/TestFrameBuilder.java

```diff
@@ -86,6 +87,54 @@ public TestFrameBuilder withVecTypes(byte... vecTypes) {
   }
 
 
+  /**
+   *  Genarate random double data for a particular column
+   * @param column for which to set data
+   * @param size size of randomly generated column
+   * @param min minimal value to generate
+   * @param max maximum value to generate
+   */
+  public TestFrameBuilder withRandomIntDataForCol(int column, int size, int min, int max) {
+    assert max > min;
+    double[] arr = new double[size];
+    for(int i = 0; i < size; i++) {
+      arr[i] = min + new Random().nextInt(max - min);
```

Comment:
a good practice is to set the seed in tests - for reproducibility

Reply:
fixed